### PR TITLE
reduced function call for get all and some other employee functions

### PR DESCRIPTION
**Pull request by ClarkeBenB**

### CODE REVIEW

**Comment by GithuCarlos** [comment]
Everything Looks Good.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # backend
 
 Server for software engineering
+
+Here is a link for endpoint testing routes in Postman, currently only supports employee and product endpoints, please read comments for each before you run it, many need id for path or values in body changed.
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/63c43057fc5bf88a2a34)
```
Comment (jaredramirez) [inline README.md]: This is pretty cool, I didn't know postman could do this!

**Comment by jaredramirez** [comment]
Much better solution than using the `sanitize` function, nice work

File: README.md (hunk 1 above)
Comment (ClarkeBenB) [reply README.md]: Yeah, I thought it was awesome, it may not be useful, but I thought it was awesome. lol

**Pull request merged**